src/helpers/misc/Globals.re

```diff
@@ -2,8 +2,22 @@
 include Belt;
 
 // From here: https://dev.to/johnridesabike/optional-chaining-in-reason-1im6
-let (<$>) = Belt.Option.map;
-let (>>=) = Belt.Option.flatMap;
+let map = Belt.Option.map;
+let (<$>) = map;
+let flatMap = Belt.Option.flatMap;
+let (>>=) = flatMap;
+let mapd = Option.mapWithDefault;
+let setDefault = (optionalValue, default) =>
+  Option.mapWithDefault(optionalValue, default, b => b);
+let (||||) = setDefault;
+
+let (|+|) = (a, b) => a->BN.addGet(. b);
+let (|*|) = (a, b) => a->BN.mulGet(. b);
+let (|-|) = (a, b) => a->BN.subGet(. b);
+let (|/|) = (a, b) => a->BN.divGet(. b);
+let (|==|) = (a, b) => a->BN.eqGet(. b);
+let (|>|) = (a, b) => a->BN.ltGet(. b);
+let (|<|) = (a, b) => a->BN.gtGet(. b);
```

Comment:
@DenhamPreen we made some changes to this file.

On further thought, `map` and `flatMap` should probably be renamed to the following:
```
let oMap = Belt.Option.map;
let flatOMap = Belt.Option.flatMap;
```

`map` and `flatMap` are very common functions in functional programming, so adding the `o` makes it explicit that they are 'optional maps'.

We can also globaly define `bnAdd`, `bnMul` etc for `|+|`, `|*|` etc, which might be more digestible for new comers to the codebase.

Reply:
Hmm I'm on the edge, one side of me says cool its much neater but the other side of me says everytime something is aliased that we are creating our own new language which makes it even more difficult to follow. Maybe a good mix in between is code documentation that has a list of all the aliases

Reply:
Great, lets do that. And start improving our documentation